plan.md

```diff
@@ -1,9 +1,282 @@
++ Building sales through delivering great work and sharing "success stories" (case studies)
++ Creating a simple way for people to share our services
+
+
+#### Metrics to Track in Services
+
++ Did we get a good case study quote from the client/project?
++ Quantitative feedback from clients (particularly if they would work with us again)
++ Sources of new leads (using Google Analytics and conversations, recording
+  this in our CRM)
+  + Number of new leads from recommendations of existing clients
++ Effectiveness of and time spent on individual marketing channels
+(particularly social media)
++ Cost of new client acquisition
++ Net promoter score (NPS)
+
+
+
+### Training
+
+Training is how we share our knowledge with individuals and organisations in our team and community, building them up.
+
+#### Why?
+**Internally**, we want to ensure dwylers continue to learn and grow; facilitating
+that process by fostering the sharing of knowledge is essential to us as an
+organisation and the learning culture we want to encourage.
+
+**Externally**, we have identified a clear need for user-centric technical training
+both based on requests from our immediately local community (particularly in the
+social good sector) and the engagement with our [existing online tutorials](https://github.com/dwyl/the-book/issues/69) and
+as evidenced by the number of training platforms and companies appearing weekly.
+
+#### How?
+
+We will use **our own products** across all of our training, demonstrating that
+we have successfully implemented what we are teaching consistently.
+
+As we do now, we will continue to evaluate each part our product and services
+cycle as a potential candidate for a learning resource and workshop.
+
+##### Workshops
+Workshops will be run *internally* in QuietSpace to share our knowledge with our
+team and with [QuietSpace](#co-working) members as well as for our [services
+clients](#services).
+
+They will be captured on video and edited so they can be shared with our remote
+community.
+
+Current workshop topics:
++ HTML/CSS for designers
++ User testing
++ [New stack](https://github.com/dwyl/technology-stack)
++ Automated testing
++ Pragmatic Agile development
+
+> Each workshop will have a defined structure and proposed audience.
+
+Initial incarnations of the 'HTML/CSS for designers', 'Pragmatic Agile
+Development' and 'Automated Testing' workshops have already been held and will
+be reviewed and iterated on.
+
+
+##### Video
+Whilst text-based tutorials have been our current way of disseminating knowledge,
+these are not great for visual learners and limit our audience to people who are
+comfortable using and navigating GitHub, which a lot of beginners are not.
+
+Expanding into video content will allow us to:
++ Establish a new channel of learning for our community (especially for visual learners)
++ Uphold our values of creating a learning community
++ Enhance our credibility as subject experts
```

Comment:
Credibility is important to dwyl as an organisation, but I'll concede it's mostly a happy side effect of the great work we do, not something we're out for specifically. I pointed it out specifically here because it's a great advantage to dwylers - working in an organisation with sme status conveys credibility to the individual 👍